docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
+- New projects are created with a default priority level and are not active
+- Projects do not have a start or end date (start or end dates are specified on allocations, users and attributions)
+- New projects must have a name, description, project owner and institution
+ 
+### Project administration
+- Principal investigators can associate new/existing users to an existing projects by email address
```

Comment:
Currently we expect the Technical Lead to share their project code with their new users so that the user can login, apply to join the project, and then the Tech Lead can approve (or not).

Reply:
I think we should support both mechanisms, I will schedule this in for the second major release.

Reply:
I think it's mostly all there, fairly minor change for a pretty nice usability boost. Agreed that it's very much a second release thing.